src/aioftp/client.py

```diff
@@ -483,9 +603,19 @@ def parse_list_line_unix(self, b):
             i = -2 if link_dst[-1] == "'" or link_dst[-1] == '"' else -1
             info["type"] = "dir" if link_dst[i] == "/" else "file"
             s = link_src
-        return pathlib.PurePosixPath(s), info
+        return pathlib.PurePosixPath(s), UnixInfoDict(
+            {
+                "type": info["type"],
+                "size": info["size"],
+                "modify": info["modify"],
+                "unix.mode": info["unix.mode"],
+                "unix.links": info["unix.links"],
+                "unix.owner": info["unix.owner"],
+                "unix.group": info["unix.group"],
+            },
+        )
```

Comment:
Why?

Reply:
It's not enabled now, but ruff has a rule from flake8 https://docs.astral.sh/ruff/rules/unnecessary-assign/

Reply:
Because of readability. If it is a `return 1, "a", 3.13` then it is ok, but when it grows it looses context, so intermediate values should be named to help the reader. Also, this helps for debugging.

Reply:
Done